UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/CirculatorySystemBase.cs

```diff
@@ -48,5 +48,20 @@ public void AddFreshBlood(ReagentMix bloodPool, float amount)
 			bloodToAdd.Add(CirculatedReagent, bloodType.GetGasCapacity(bloodToAdd));
 			bloodPool.Add(bloodToAdd);
 		}
+
+		public void Bleed(float amount)
+		{
+			var bloodToRemove = new ReagentMix(BloodType, amount);
+			bloodToRemove.Remove(CirculatedReagent, bloodType.GetGasCapacity(bloodToRemove));
+			ReadyBloodPool.Subtract(bloodToRemove);
```

Comment:
you don't need to subtract the blood component, you can just remove chemicalswhatever u amount of chemicals

Reply:
I don't understand.

Reply:
done.